addons/crm/models/utm.py

```diff
@@ -6,37 +6,25 @@
 class UtmCampaign(models.Model):
     _inherit = 'utm.campaign'
 
-    crm_lead_activated = fields.Boolean('Use Leads', compute='_compute_crm_lead_activated')
-    lead_count = fields.Integer('Lead Count', groups='sales_team.group_sale_salesman', compute="_compute_global_opportunity_and_lead_count")
-    opportunity_count = fields.Integer('Opportunity Count', groups='sales_team.group_sale_salesman', compute="_compute_global_opportunity_and_lead_count")
+    use_leads = fields.Boolean('Use Leads', compute='_compute_use_leads')
+    leads_opportunities_count = fields.Integer('Leads/Opportunities count', groups='sales_team.group_sale_salesman', compute="_compute_leads_opportunities_count")
```

Comment:
I propose to name it crm_lead_count to have shorter name (and more coherent with model itself). What do you think ?